tests/mail/tests.py

```diff
@@ -1157,6 +1181,7 @@ def test_sanitize_address(self):
             (
                 "Tó Example <tó@example.com>",
                 "utf-8",
+                # (Not RFC-2047 compliant.)
```

Comment:
```
                # (Not RFC-2047 compliant.)
```
I think all three are not RFC-2047 compliant (from below `# An 'encoded-word' localpart...`) so I don't think we need to call it out here

Reply:
(I rearranged the cases slightly to group all of the non-compliant ones together.)